Ariana_Abdo/main.js

```diff
@@ -1,29 +1,42 @@
-document.addEventListener('DOMContentLoaded', render);
+ЗАДАЧА №1
+function createArray(size) {
+    return [...new Array(size)].map((v,k) => k);
+}
 
-const text = {
-    header: 'This is the file for your home tasks',
-    list: [
-        'Please, write your JavaScript code in <strong>main.js</strong>',
-        'Please, write your CSS code in <strong>styles.css</strong>',
-        'Please, write your HTML right here in <strong>index.html</strong>'
-    ]
-};
+const arr = createArray(100);
+for (let i = 1; i < arr.length; i++){
+    
+    if ((arr[i] % 2) == 0 ){
+        console.log(arr[i]);
+    }
+}
 
-function render() {
-    const main = document.createElement('main');
-    const h2 = document.createElement('h2');
-    h2.textContent = text.header;
+ЗАДАЧА №2
+function createArray(size) {
+    return [...new Array(size)].map((v,k) => k);
+}
 
-    const ul = text.list.reduce((ul, text) => {
-        const li = document.createElement('li');
+const arr = createArray(100);
+for (let i = 99; i < arr.length; i--){
```

Comment:
why is the counter is hard-coded here? what if we'll take another array, for 200 items, for example? would it work?
Also, the condition when `i < arr.length` would not work here properly.

Here you need to make the reverse direction. One of ways to do it - just need to define the counter with arr.length value, and the limit value - use zero (0).

```suggestion
for (let i = arr.length - 1; i >= 0; i--) {...}
```